modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -641,6 +622,25 @@ export default class I3SConverter {
       transform: transformationMatrix,
       parentNodes: childNodes
     };
+
+    if (sourceTile.id) {
+      this.progresses[PROGRESS_PHASE1_COUNT].stepsDone += 1;
+
+      let timeRemainingString = 'Calculating...';
+      const timeRemainingObjectBasedOnCount =
+        this.progresses[PROGRESS_PHASE1_COUNT].getTimeRemaining();
+      if (timeRemainingObjectBasedOnCount !== null && timeRemainingObjectBasedOnCount.trust) {
+        timeRemainingString = `${timeRemainingObjectBasedOnCount.timeRemaining.toFixed(0)} seconds`;
+      }
+
+      let percentString = this.progresses[PROGRESS_PHASE1_COUNT].getPercentString();
+      if (percentString) {
+        percentString = ' ' + percentString;
+      }
+      console.log(
+        `[converted${percentString}%, time remaining = ${timeRemainingString}]: ${sourceTile.id}`
```

Comment:
As discussed:
```suggestion
        `[converted${percentString}%, ${timeRemainingString} left]: ${sourceTile.id}`
```

Reply:
Done, but removed source.id from the "converted" string